config/passport.js

```diff
@@ -10,14 +10,29 @@ passport.use(
             passwordField: "user[password]"
         },
         function(email, password, done) {
-            User.findOne({ email: email })
+            User
+            .findOne({
+                where: {
+                    email: email
+                }
+            })
```

Comment:
@chidioguejiofor can you use object destructing in the email field to avoid hound shouting.

``` 
where: {
  email,
}
```
will be ok